docs/research_results.md

```diff
@@ -0,0 +1,82 @@
+Trying to guess how long it could take to build the network I read [this](https://hackernoon.com/learning-keras-by-implementing-vgg16-from-scratch-d036733f2d5)
+and I would say we should be able to build the network in not more then 3 weeks(a 10 hours/person per week).
+If the people are a little bit used to tensorflow and nn's it could be done in half of the time.
+I think its essential, because all this papers let assume that plan the training, seperate the datasets, prepare them and train the models will take a lot more time.
+We should talk with Prof. Hartung about using pretrained weights with our planned vgg-16 network.
+The [team](https://www.researchgate.net/publication/283356929_DEX_Deep_EXpectation_of_Apparent_Age_from_a_Single_Image) which used vgg-16 used an model pretrained on the net datasets
+on general classification. This would improve our result enormous because it would take up to 5 weeks on 4 gpu's wthout a pretrained vgg-16 and we don't have the time for that we need to use pretrained weights, choose
+another nn or accept bad results.
+
+Next steps I would choose:
+- __talk to Prof Hartung about using pretrained weights(otherwise search for other fitting net)__
+- research if vgg-16 is a good choice for gender estimation too
+- understanding what convolutional nn's exactly do
+- image group:
+  - which iput vgg-16 takes
+  - how to split the datasets
+  - how to prepare the images(rotate or zoom them for training => better generalization)
+  - get datasets
+- nn group:
+  - build the network
+  - get in touch with tensorboard
+  - play arround with smaller nn's and learn how to train(all the stuff with graphs and optimization)
+
+
+<p>
+title: Alex Notizen<br>
+author: Alex Wiens<br>
+date: 13.11.18
+</p>
+
+VGG-16 erkennt zwar gut alter und Geschlecht, für weitere Features ist es aber vielleicht sinnvoll erstmal einen Gesichts-ausschnitt zu erhalten (eventuell auch mit Augen Position um je nachdem nachher einfacher die Augenfarbe zu erhalten).
+Hierfür bietet sich ein einfaches NN basierend auf "Haar-Cascades" an (also relativ einfache Kontrast Analyse zwischen verschiedenen Positionen im Bild) welches von Intel zur Verfügung gestellt wird und direkt mit OpenCV ausgeliefert wird (allerdings als separate XML Datei)
+
+- https://github.com/shantnu/FaceDetect/blob/master/face_detect.py
+- https://realpython.com/face-detection-in-python-using-a-webcam/
+
+
+__english version:__
+VGG-16 is good for age and gender recognition, but for further features it might be useful to get face detail first (possibly with eye position to get the eye color easier afterwards). For this a simple NN based on "hair cascades" (i.e. relatively simple contrast analysis between different positions in the image) which is provided by Intel and shipped directly with OpenCV (but as a separate XML file** is useful.
```

Comment:
```suggestion
VGG-16 is good for age and gender recognition, but for further features it might be useful to get a face/face detail cutout first (possibly with eye position to get the eye color easier afterwards). For this a simple NN based on "Haar cascades" (i.e. relatively simple contrast analysis between different positions in the image) which is provided by Intel and shipped directly with OpenCV (but as a separate XML file** is useful.
```